common-rendering/src/components/figCaption.tsx

```diff
@@ -5,49 +5,57 @@ import { css } from '@emotion/react';
 import { remSpace } from '@guardian/source-foundations';
 import { brandAltText, neutral, text } from '@guardian/source-foundations';
 import { textSans } from '@guardian/source-foundations';
+import { SvgCamera } from '@guardian/source-react-components';
 import type { ArticleFormat } from '@guardian/libs';
 import { ArticleDesign } from '@guardian/libs';
 import type { Option } from '@guardian/types';
 import { OptionKind } from '@guardian/types';
 import type { FC, ReactNode } from 'react';
-import { fill } from '../editorialPalette';
 import { darkModeCss } from '../lib';
 
 // ----- Sub-Components ----- //
 
-interface TriangleProps {
+interface CameraProps {
 	format: ArticleFormat;
 	supportsDarkMode: boolean;
 }
 
-const triangleStyles = (
-	format: ArticleFormat,
-	supportsDarkMode: boolean,
-): SerializedStyles => css`
-	fill: ${fill.icon(format)};
-	height: 0.8em;
-	padding-right: ${remSpace[1]};
+const cameraStyles = (supportsDarkMode: boolean): SerializedStyles => css`
+	display: inline-block;
+	width: 1.2rem;
+	margin-right: ${remSpace[1]};
+	fill: ${text.supporting};
+	position: relative;
+
+	::before {
+		content: ' ';
+		display: block;
+		padding-top: 0.85rem;
+	}
+
+	svg {
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+	}
 
 	${darkModeCss(supportsDarkMode)`
-        fill: ${fill.iconDark(format)};
+        fill: ${brandAltText.supporting};
```

Comment:
I think `text` and `brandAltText` were deprecated in v4 of `source-foundations`? The closest available I can find in the current version are:

- `text.supporting` -> `neutral[46]`
- `brandAltText.supporting` -> `neutral[60]`

You may want to add these to the `fill` module in `editorialPalette`, similar to how the fill colour is managed in the DCR example you linked above. Perhaps `cameraCaptionIcon` and `cameraCaptionIconDark`? I think, in general, if you're setting a colour (via `color`, `background-color`, `fill`, `border` etc.) it's worth considering whether it could live in `editorialPalette`.

Reply:
Oh yes, `text` and `brandAltText` are deprecated! Strangely, VSCode wasn't marking the imports as deprecated - I think because the version of `source-foundations` used hadn't added the JSDoc yet.

That's a great point about adding these colours to `editorialPalette`, thanks!